src/preact/storybook/Context.js

```diff
@@ -1,36 +1,37 @@
-// @ts-ignore
-import {boolean, select, withKnobs} from '@storybook/addon-knobs';
-
 import {Loading_Enum} from '#core/constants/loading-instructions';
 
 import * as Preact from '#preact';
 import {WithAmpContext, useAmpContext, useLoading} from '#preact/context';
 
 export default {
   title: '0/Context',
-  decorators: [withKnobs],
 };
 
 const IMG_SRC =
   "data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24px' height='24px' viewBox='0 0 24 24' fill='%23000000'%3E%3Cpath d='M0 0h24v24H0z' fill='none'/%3E%3Cpath d='M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z'/%3E%3C/svg%3E";
 
-const LOADING_OPTIONS = ['auto', 'lazy', 'eager', 'unload'];
-
-export const _default = () => {
```

Comment:
FMI: why is this named _default? Does that do something?

Reply:
As a convention we name the first story `_default` to make it clear its one that gets default selected when you navigate to the story. It doesnt do anything special in the storybook its more just for us 

the `default` on line 9/6 That one does some storybook internal magic and is important to be named that way